core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -380,12 +380,14 @@
 
   final ThreadPoolExecutor mSyncPrefetchExecutor = new ThreadPoolExecutor(
       ServerConfiguration.getInt(PropertyKey.MASTER_METADATA_SYNC_UFS_PREFETCH_POOL_SIZE),
```

Comment:
Can you change the value of `PropertyKey.Name.MASTER_METADATA_SYNC_UFS_PREFETCH_POOL_SIZE` to meet the prop name. It currently ends with `parallelism`.

Reply:
updated